### ISSUE TITLE
Configure all the repositories

### ISSUE THREAD

**Issue by daenney** [etsy/Hound#13]
It would be really helpful if you could just tell it to search/index all the repositories under a namespace. With some organisations having hundreds of repo's needing to explicitly define every single one of them is needless work.

If it's only for repo's on Github it could be automated through the Github API.

**Comment by jklein** [comment]
Yeah this is pretty GitHub specific, but shouldn't be too hard to add. We would welcome the contribution if you want to take a stab at it :-)

**Comment by daenney** [comment]
Do you have a suggestion how to do this? The wackiest thing I can come up with is a simple cron that just polls the Github API for your organisation and templates out the configuration.

I'm pretty sure we can come up with something better though.

**Comment by jklein** [comment]
Yeah I'd probably prefer to have the organization in the config file, and just have the code that pulls figure that out and clone/index all repos under it using the GitHub API. Since it's so GitHub specific I would just suggest adding another key in the config file called `organizations` or something that's parallel to `repos`.

**Comment by daenney** [comment]
Right, that makes more sense indeed. Time to try my hand at Go I guess :smile:.

**Comment by Napolskih** [comment]
I just need this feature.
:+1:

**Comment by bmoyles** [comment]
I might suggest making repo discovery more generic, possibly pluggable (maybe something similar to how Packer does plugins) for orgs that are using something like Github for Enterprises, Stash, Bitbucket, etc.

**Comment by pastjean** [comment]
:+1: for @bmoyles idea 
even directory file globs for local files

**Comment by sstarcher** [comment]
+1

**Comment by sstarcher** [comment]
#158 does some of this

**Comment by metasyn** [comment]
This is a great idea
